milli/src/documents/builder.rs

```diff
@@ -181,6 +202,7 @@ fn parse_csv_header(header: &str) -> (&str, AllowedType) {
     match header.rsplit_once(':') {
         Some((field_name, field_type)) => match field_type {
             "string" => (field_name, AllowedType::String),
+            "boolean" => (field_name, AllowedType::Boolean),
```

Comment:
It impacts the documentation team, please warn them 😊

Reply:
Done! I sent them a ping in the issue 👍